Dockerhub rate limit: rename add-on to app (#43311)
home-assistant/home-assistant.github.io: source/more-info/dockerhub-rate-limit.markdown

To use your Docker Hub credentials with the Supervisor:

1. You need to have the advanced user toggle enabled in your user profile setting.

> 2. Go to the {% my supervisor title="**Settings** > **App**" %} in the Home Assistant UI.

Author: ```suggestion
2. Go to the {% my supervisor title="**Settings** > **Apps**" %} in the Home Assistant UI.
```

> 2. Go to the {% my supervisor title="**Settings** > **Apps**" %} in the Home Assistant UI.

Reviewer: Actually just realized this redirects to the old Supervisor panel.

From the [my redirect list](https://github.com/home-assistant/my.home-assistant.io/blob/main/redirect.json#L175-L180) it seems that this particular my link is actually deprecated.

I could not find a replacement my link which goes to the installed apps. But there is Apps store redirect with `supervisor_store`. But you'd need to conflate 2. and 3. here to use that link.

> 2. Go to the {% my supervisor title="**Settings** > **Apps**" %} in the Home Assistant UI.

Author: ```suggestion
2. Go to {% my supervisor_store title="**Settings** > **Apps** > **Install app**" %}.
```

> 3. Select **Install app**.

Author: ```suggestion
```

> 4. In the top-right corner of the screen, select the three dots {% icon "mdi:dots-vertical" %} menu, and select **Registries**.

Author: ```suggestion
3. In the top-right corner of the screen, select the three dots {% icon "mdi:dots-vertical" %} menu, and select **Registries**.
```

> 5. In the dialog that opens up, select **Add new registry** and enter `docker.io` as the registry followed by your credentials:

Author: ```suggestion
4. In the dialog that opens up, select **Add new registry** and enter `docker.io` as the registry followed by your credentials:
```

> 5. In the dialog that opens up, select **Add new registry** and enter `hub.docker.com` as the registry followed by your credentials:

Reviewer: While at it, hub.docker.com is actually the legacy domain for Docker Hub. Users should use `docker.io` nowadays:

```suggestion
5. In the dialog that opens up, select **Add new registry** and enter `docker.io` as the registry followed by your credentials:
```

See also: https://github.com/home-assistant/supervisor/pull/6360
